### ISSUE TITLE
Attempt to use identical GSL versions in wheels.

### ISSUE THREAD

**Issue by benjeffery** [tskit-dev/msprime#1655]
In https://github.com/tskit-dev/tskit/pull/1312 we found that OSX is giving different results to Linux when msprime is conda-installed, likely due to differing GSL versions. @jeromekelleher has also noticed that this happens with the wheels, likely as they are built on the same conda install.

If possible it would be good to get all the wheels on the same GSL version.

**Comment by jeromekelleher** [comment]
I don't think this interacts with conda at all - we're installing GSL from brew on OSX and we're downloading it from source in the buildwheel linux script. So should be easy enough to pin to a conservative version that's available on brew and get that version from source too. We don't need anything super recent, we're only using the rng stuff which is super stable.

**Comment by jeromekelleher** [comment]
Hmmm, so we're actually using GSL 2.6 on both linux and OSX. For linux, we build 2.6 from source and the version that's in homebrew is also 2.6. I don't know why mac and linux would be giving different values, outputs, then.

Since the specific issue isn't actually the problem, I'm going to close it.

**Comment by benjeffery** [comment]
Someone else has hit this issue at https://github.com/tskit-dev/msprime/discussions/1685#discussioncomment-752745

**Comment by jeromekelleher** [comment]
I agree we should get to the bottom of why we're getting different random streams on the different OSs, but I don't think the reason is differing GSL versions in the wheels. I'm fairly sure both wheels are using the same GSL version.

Could someone with access to a Mac and linux system verify this please? We need to (a) pip install from the binary wheel (preferably in a fresh venv);  (b) ``msp ancestry 10 > tmp.trees; tskit provenances tmp.trees`` and paste the output here.

**Comment by benjeffery** [comment]
I have an old mac hanging around somewhere, it hasn't booted OSX in years! I'll see if I can get set up on it as it would be useful to debug locally sometimes.

**Comment by jeromekelleher** [comment]
I don't think this is worth worrying about, closing
